src/epcpm/parameterstointerface.py

```diff
@@ -1410,6 +1428,11 @@ def create_item(
         include_uuid_in_item=include_uuid_in_item,
     )
 
+    maybe_sunspec_variable_length = []
+    if parameter.internal_type == "UartName":
```

Comment:
I think (maybe) this function so far doesn't have any significant type awareness?  Perhaps this goes outside and comes in more like `variable_or_getter_setter`?